src/routes/categories/containers/CategoriesContainer.js

```diff
@@ -49,5 +52,12 @@ const mapStateToProps = (state: StateType, ownProps) => {
 
 // $FlowFixMe
 const themed = withTheme(props => <ScrollView><Categories {...props} /></ScrollView>)
-// $FlowFixMe connect()
-export default withRouteCleaner(connect(mapStateToProps, mapDispatchToProps)(themed))
+export default compose( // $FlowFixMe connect()
+  withRouteCleaner,
+  connect((state: StateType, ownProps) => ({
+    invalidLanguage: !state.cityContent.categoriesRouteMapping[ownProps.navigation.getParam('key')]
```

Comment:
I think it would make sense to create a function for `ownProps.navigation.getParam('key')` at some point.